src/css/layout/ScrollbarLayout.css

```diff
@@ -57,7 +52,9 @@
   content: '';
   display: block;
   position: absolute;
-  transition: background-color 250ms ease;
+  transition-duration: 250ms;
+  transition-timing-function: ease;
+  transition-property: background-color, height, width;
```

Comment:
nit: should this be turned into a single line?

Reply:
do we have any advantage of that? 
code wise i think its fine